.github/workflows/reusable-build-flavor.yaml

```diff
@@ -3,38 +3,46 @@ name: Reusable workflow that builds a specific Kairos flavor
 on:
   workflow_call:
     inputs:
-      flavor:
-        required: true
-        type: string
-      flavor_release:
-        required: true
-        type: string
-      family:
-        required: true
-        type: string
       base_image:
         required: true
         type: string
       model:
-        required: true
+        required: false
         type: string
+        default: "generic"
       variant:
-        required: true
+        required: false
         type: string
+        default: "core"
       arch:
+        required: false
+        type: string
+        default: "amd64"
+      kubernetes_version:
+        required: false
+        type: string
+      event_type:
         required: true
         type: string
 
 permissions: read-all
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ inputs.arch == 'arm64' && 'ubuntu-24.04-arm' || 'ubuntu-24.04' }}
```

Comment:
what does this do? if arm64 it will output `ubuntu-24.04-arm` otherwise `ubuntu-24.04` ?

Reply:
yep! So each job runs in its proper runnner arch depending on its own arch.

Arm builds will run in the arm workers and amd ones in the normal ones :D